tests/installation/user_settings.pm

```diff
@@ -44,7 +44,6 @@ sub run {
         record_soft_failure('boo#1122804 - Typing issue with fullname') unless match_has_tag('inst-userinfostyped-expected-typefaces');
         $retry++;
     } while (($retry < $max_tries) && !match_has_tag('inst-userinfostyped-expected-typefaces'));
-    assert_screen('inst-userinfostyped-expected-typefaces');    # fail if mistyped
```

Comment:
This change will prevent test failing when we have mistyped the name after 3 attempts: https://openqa.suse.de/tests/3996551#step/user_settings/10 
Which will waste resource. So in case we want to act on this instability, we should increase amount of attempts to type everything.

Reply:
isnt the while loop a workaround to check if the typos still happens and raise a soft_fail? The soft_fail is there to not hide the issue. am i right?

Reply:
Yes, but idea is to fail the test if workaround didn't help which is done by the assertion you want to remove. In your VR we have typo in the name, so if something fails later will be harder to investigate, because we might assume that workaround did work.

Reply:
We don't want the test to continue if there are typos.

Reply:
Workaround failed after retries on your verification run: https://openqa.suse.de/tests/3996551#step/user_settings/10

Reply:
i am a bit confused. We knew about that typing issue. But at least it could continue and verify the installation scenario. And we had those soft_fails to not hide the issue. But if we really want to let the test fail for this, feel free to close this PR. 
I tried yesterday to increase the $max_retries without success.

Reply:
Yes, that was the case until RM's started asking for 'resolve the soft-fail'. ;)

But, as I see that this is annoying several teams, I will make a change to only fail for functional job group.

Reply:
@rwx788 are you ok with this? if so lets close this